diff --git a/src/common.h b/src/common.h
index 11401d6..e5f74da 100644
--- a/src/common.h
+++ b/src/common.h
@@ -537,8 +537,6 @@ INLINE unsigned long long rotl_64(const unsigned long long x, const unsigned int
 {
 #ifdef _MSC_VER
     return _rotl64(x, count);
-#elif !defined(__aarch64__) && (defined(__INTEL_COMPILER) || (defined(__GNUC__) && (__GNUC__ > 3)) || defined(__clang__))
-    return __rolq(x, count);
 #else
     return (x<<count) | (x>>( (unsigned int)(-(int)count)&63 )); // -count&63 instead of 64-count to handle count==0
 #endif
