diff --git a/src/common.h b/src/common.h
index e5f74da..9d7c587 100644
--- a/src/common.h
+++ b/src/common.h
@@ -557,8 +557,6 @@ INLINE unsigned long long rotr_64(const unsigned long long x, const unsigned int
 {
 #ifdef _MSC_VER
     return _rotr64(x, count);
-#elif !defined(__aarch64__) && (defined(__INTEL_COMPILER) || (defined(__GNUC__) && (__GNUC__ > 3)) || defined(__clang__))
-    return __rorq(x, count);
 #else
     return (x>>count) | (x<<( (unsigned int)(-(int)count)&63 )); // -count&63 instead of 64-count to handle count==0
 #endif
